src/elements/content-sidebar/SidebarContent.js

```diff
@@ -36,6 +37,7 @@ const SidebarContent = ({ actions, children, className, title, elementId, sideba
             <div className="bcs-scroll-content-wrapper">
                 <div className="bcs-scroll-content">{children}</div>
             </div>
+            {overlay}
```

Comment:
Can we encapsulate the new UI elements so as to not require a top-level overlay? I don't think this is going to be a reused experience.